login-fire-common-behavior.html

```diff
@@ -209,6 +209,17 @@
       },
 
       /**
+       * Automatically sign in the user with credentials stored in
+       * Credentials API
+       *
+       * @type {Boolean}
+       */
+      autoLogin: {
```

Comment:
What about rename this property "auto"?

_IMHO, `login-fire` elements have for main purpose to log-in._

Reply:
Legit :+1: